src/main/scala/com/atomist/rug/runtime/js/interop/jsSafeCommittingProxy.scala

```diff
@@ -210,7 +215,7 @@ object jsSafeCommittingProxy {
     * @param nodes sequence to wrap
     * @return TypeScript and JavaScript-friendly list
     */
-  def wrap(nodes: Seq[GraphNode], typeRegistry: TypeRegistry): java.util.List[jsSafeCommittingProxy] =
+  def wrap(nodes: Seq[GraphNode], typeRegistry: TypeRegistry): java.util.List[GraphNode] =
     new JavaScriptArray(nodes.map(wrapOne(_, typeRegistry)).asJava)
 
   def wrapIfNecessary(nodes: Seq[_], typeRegistry: TypeRegistry): java.util.List[Object] = {
```

Comment:
This is being passed to a Java list, so I think it makes sense to be explicit about the Java type